inlang/source-code/create-project/src/create.ts

```diff
@@ -1,52 +1,62 @@
 import type { ProjectConfig } from "@inlang/project-config"
 import type { NodeishFilesystem } from "@lix-js/fs"
-import { tryAutoGenModuleConfig } from "./tryAutoGenModuleConfig.js"
+import { tryAutoGenProjectConfig } from "./tryAutoGenProjectConfig.js"
+import { openInlangProject } from "@inlang/sdk"
+import { tryCatch } from "@inlang/result"
 
 /**
  * Creates a now project.inlang.json file.
  *
  * @params args.nodeishFs a nodeishFs implementation, in node this is fs:promieses
  * @params args.pathJoin implementation for joing paths, in node this should be {join} from 'path'
  * @params args.filePath optional location of your inlang config file, should be used only for testing as we hard code filepaths at lots of places
- * @params args.tryAutoGen if the module configuration should be heuristically tried to generate, if false this will be only a minimal config file
- * @params args.languageTags the languages that should be supported next to the source language
- * @params args.sourceLanguagetag the main language that other langauges are translated to
  * @returns The warnings and if successfully generated the config object, the file is also saved to the filesystem in the current path
  */
 export async function createProjectConfig(args: {
```

Comment:
I would delete `createProjectConfig` and place all logic in `tryAutoGenProjectConfig` to be explicit in what the function does: It tries to generate a project config, not to create a project.